content/applications/sales/sales/products_prices/prices/pricing.rst

```diff
@@ -1,146 +1,404 @@
+- :guilabel:`Discount`: the price computation is based on a discount.
+- :guilabel:`Formula`: the price computation is based on a formula.
 
-.. image:: pricing/formula_discount.png
+.. note::
+   Each option reveals its own computation-specific fields on the form.
+
+If :guilabel:`Fixed Price` is selected, enter the desired price in the :guilabel:`Fixed Price`
+field below. If :guilabel:`Discount` is selected, enter the desired discount percentage in the
+:guilabel:`Discount` field that appears.
+
+If :guilabel:`Formula` is selected, a number of configurable options appear.
+
+.. image:: pricing/formula-computation-options.png
    :align: center
+   :alt: The various formula computation options present in Odoo Sales.
+
+With the :guilabel:`Formula` computation option selected, start by selecting an option from the
+:guilabel:`Based on` field: :guilabel:`Sales Price`, :guilabel:`Cost`, or :guilabel:`Other
+Pricelist`. This determines what the advanced price rule formula will be based on.
 
-Costs with markups (retail)
----------------------------
+Next, in the :guilabel:`Discount` field, determine how much of a discount should be applied. It
+should be noted that a mark-up can be applied by setting a negative discount in this field.
 
-e.g. sale price = 2*cost (100% markup) with $5 of minimal margin.
+.. example::
+   Here's an example of what it would like to formulate `2 x Cost` (100% markup), with $5 minimal
+   margin using an advanced pricelist rule formula. This is often seen in retail situations.
 
-.. image:: pricing/formula_cost.png
+.. image:: pricing/formula-markup-cost-example.png
    :align: center
+   :alt: How it looks to formulate a markup cost with 5 dollar minimum margin in Odoo Sales.
 
-Prices per country
-==================
+Then, in the :guilabel:`Extra Fee` field, specify a fixed amount to add or subtract (if negative)
+to the amount calculated with the discount. After that, enter a desired figure in the
+:guilabel:`Rounding Method` field. This sets the price, so that it is a multiple of the value in
+the field.
 
-Pricelists can be set by countries group. Any new customer recorded in Odoo gets a default
-pricelist, i.e. the first one in the list matching the country. In case no country is set for the
-customer, Odoo takes the first pricelist without any country group.
+.. note::
+   Rounding is applied *after* the discount and *before* the surcharge.
+
+.. tip::
+   To have prices that end in 9.99, set rounding to `10`, surcharge `-0.01`.
+
+Lastly, specify the minimum amount of margin over the base price in the :guilabel:`Margins` field.
+
+Once all formula-related configurations are complete, Odoo provides an example of the formula in
+a blue block to the right of the configurations.
 
-The default pricelist can be replaced when creating a sales order.
+.. example::
+   Here's how it would look to apply 20% discounts, with prices rounded up to 9.99.
```

Comment:
Same here, the text should describe the entire configuration, and not directly reference the image by saying "here's how it would look."